### PR TITLE
[Fluent] Imageeditor styles

### PR DESCRIPTION
**Pull request by kikostadinov**
Related to: https://github.com/telerik/kendo-themes-private/issues/3

### CODE REVIEW

File: packages/fluent/scss/index.scss [hunk 1]
```diff
@@ -134,8 +134,8 @@
 
 
 // Editors
-@use "editor";
-// @use "imageeditor";
+// @use "editor";
```
Comment (Juveniel) [inline packages/fluent/scss/index.scss:137]: uncomment this

File: packages/fluent/scss/index.scss [hunk 2]
```diff
@@ -296,8 +296,8 @@
 
 
     // Editors
-    @include editor.styles();
-    // @include imageeditor.styles();
+    // @include editor.styles();
```
Comment (Juveniel) [inline packages/fluent/scss/index.scss:299]: uncomment this

File: packages/fluent/scss/imageeditor/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,47 @@
+@use "../core/_variables.scss" as *;
+@use "../core/color-system" as *;
+
+// ImageEditor
+$kendo-imageeditor-margin-y: null !default;
+$kendo-imageeditor-margin-x: null !default;
+$kendo-imageeditor-padding-y: null !default;
+$kendo-imageeditor-padding-x: null !default;
```
Comment (Juveniel) [inline packages/fluent/scss/imageeditor/_variables.scss:8]: set 0px

File: packages/fluent/scss/imageeditor/_variables.scss [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+$kendo-imageeditor-action-pane-bg: var( --kendo-component-bg, inherit ) !default;
+$kendo-imageeditor-action-pane-text: null !default;
+$kendo-imageeditor-action-pane-border: null !default;
+
+$kendo-imageeditor-crop-bg: null !default;
+$kendo-imageeditor-crop-text: null !default;
+$kendo-imageeditor-crop-border: white !default;
+
+$kendo-imageeditor-crop-overlay-bg: rgba(black, .3) !default;
```
Comment (Juveniel) [inline packages/fluent/scss/imageeditor/_variables.scss:47]: spacing

**Review by Juveniel — CHANGES_REQUESTED** [review]

**Review by TeyaVes — APPROVED** [review]

**Review by Juveniel — APPROVED** [review]

**Pull request merged**